index.d.ts

```diff
@@ -49,7 +49,7 @@ export type Spec<T, C extends CustomCommands<object> = never> =
   | (C extends CustomCommands<infer O> ? O : never);
 
 type ArraySpec<T, C extends CustomCommands<object>> =
-  | { $push: T }
+  | { $push: Array[T] }
```

Comment:
Looks like you need to use `T[]` instead

Reply:
ah yes sorry!

Reply:
You can just make another commit with the change and push it to your `patch-1` branch it should be reflected in this PR

Reply:
My mistake, I somehow missed that push takes an array, not an item. This should indeed be `T[]`.

Please make sure the tests are updated to reflect this change!